language/move-prover/spec-lang/src/lib.rs

```diff
@@ -56,7 +56,7 @@ pub fn run_spec_lang_compiler(
     // First pass: compile move code.
     let (files, units_or_errors) = move_compile_no_report(&all_sources, &[], address_opt)?;
     // Enter sources into env, remember file ids as
-    for (fname, fsrc) in files {
+    for (fname, fsrc) in itertools::sorted(files.iter()) {
```

Comment:
Would you mind sorting only the keys? The second pair of the pair is actually the entire source text.

```
 for fname in files.keys().sorted() {
        let fsrc = files.get(fname);
        ....
```

Reply:
Done.